Wagr/Application/SettingsTab/SettingsViewController.m

```diff
@@ -108,7 +108,7 @@ - (IBAction)onLogoutBtnPressed:(id)sender {
     
 
     // We need the enrolled account UPN for both the SDK and MSAL
-    NSString *enrolledAccount = [IntuneMAMEnrollmentManager instance].enrolledAccount;
+    NSString *enrolledAccount = [IntuneMAMEnrollmentManager instance].enrolledAccountId;
```

Comment:
Will the call to MSALPublicClientApplication accountForUsername on line 127 work if we give it an OID?